### PR TITLE
Quality: PowerShell Summit GH-3391

### LINKED ISSUE #3391: Quality: Install-ADDSForest.md style guidlines
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Aliases, Formatting code samples, Formatting command syntax, Link References, Markdown linting, Spelling

### Article List

- Install-ADDSForest.md

### PR DESCRIPTION
**Pull request by RobBiddle**
# PR Summary
- Resolves #3391 

## PR Checklist
- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

### CODE REVIEW

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 1]
```diff
@@ -11,44 +11,65 @@ title: Install-ADDSForest
...
-PS C:\> Install-ADDSForest -DomainName "corp.contoso.com" -InstallDNS
+
+```powershell
+Install-ADDSForest -DomainName "corp.contoso.com" -InstallDNS
 ```
 
-This command installs a new forest named corp.contoso.com, causes the user to be prompted to provide and confirm the Directory Services Restore Mode (DSRM) password and specifies a DNS server should also be installed during the forest installation process.
+This command installs a new forest named corp.contoso.com, causes the user to be prompted to provide
+and confirm the Directory Services Restore Mode (DSRM) password and specifies a DNS server should
+also be installed during the forest installation process.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:42]:
```suggestion
This command installs a new forest named `corp.contoso.com`, causes the user to be prompted to provide
and confirm the Directory Services Restore Mode (DSRM) password and specifies a DNS server should
also be installed during the forest installation process.
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 2]
```diff
@@ -11,44 +11,65 @@ title: Install-ADDSForest
...
+}
+Install-ADDSForest @HashArguments
 ```
 
-This command installs a new forest named corp.contoso.com, creates a DNS delegation in the contoso.com domain, sets domain functional level to Windows Server 2008 R2 and sets forest functional level to Windows Server 2008, installs the Active Directory database and SYSVOL on the D:\ drive, installs the log files on the E:\ drive and has the server automatically restart after AD DS installation is complete and prompts the user to provide and confirm the DSRM password.
+This command installs a new forest named corp.contoso.com, creates a DNS delegation in the
+contoso.com domain, sets domain functional level to Windows Server 2008 R2 and sets forest
+functional level to Windows Server 2008, installs the Active Directory database and SYSVOL on the
+`D:\` drive, installs the log files on the `E:\` drive and has the server automatically restart
+after AD DS installation is complete and prompts the user to provide and confirm the DSRM password.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:63]:
```suggestion
This command installs a new forest named `corp.contoso.com`, creates a DNS delegation in the
`contoso.com` domain, sets domain functional level to Windows Server 2008 R2 and sets forest
functional level to Windows Server 2008, installs the Active Directory database and SYSVOL on the
`D:\` drive, installs the log files on the `E:\` drive and has the server automatically restart
after AD DS installation is complete and prompts the user to provide and confirm the DSRM password.
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 3]
```diff
@@ -93,11 +117,12 @@ Accept wildcard characters: False
 ```
 
 ### -DnsDelegationCredential
-Specifies the user name and password for creating DNS delegation.
-This parameter is skipped if the value for the *CreateDnsDelegation* parameter is either specified or computed to be $False.
+
+Specifies the user name and password for creating DNS delegation. This parameter is skipped if the
+value for the `-CreateDnsDelegation` parameter is either specified or computed to be $False.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:122]:
```suggestion
Specifies the user name and password for creating DNS delegation. This parameter is skipped if the
value for the **CreateDnsDelegation** parameter is either specified or computed to be `$false`.
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 4]
```diff
@@ -109,9 +134,11 @@ Accept wildcard characters: False
...
 
 ### -DomainMode
-Specifies the domain functional level of the first domain in the creation of a new forest.
-Supported values for this parameter can be either a valid integer or a corresponding enumerated string value.
-For instance, to set the domain mode level to Windows Server 2008 R2, you can specify either a value of 4 or Win2008R2.
+
+Specifies the domain functional level of the first domain in the creation of a new forest. Supported
+values for this parameter can be either a valid integer or a corresponding enumerated string value.
+For instance, to set the domain mode level to Windows Server 2008 R2, you can specify either a value
+of 4 or Win2008R2.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:141]:
```suggestion
Specifies the domain functional level of the first domain in the creation of a new forest.
Supported values for this parameter can be either a valid integer or a corresponding enumerated
string value. For instance, to set the domain mode level to Windows Server 2008 R2, you can specify
either a value of `4` or `Win2008R2`.
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 5]
```diff
@@ -154,19 +182,24 @@ Accept wildcard characters: False
...
-For NetBIOS names to be valid for use with this parameter they must be single label names of 15 characters or less.
 
-If this parameter is set with a valid NetBIOS name value, then forest installation continues with the name specified.
-If this parameter is not set, then the default is automatically computed from the value of the *DomainName* parameter.
+Specifies the NetBIOS name for the root domain in the new forest. For NetBIOS names to be valid for
+use with this parameter they must be single label names of 15 characters or less.
+
+If this parameter is set with a valid NetBIOS name value, then forest installation continues with
+the name specified. If this parameter is not set, then the default is automatically computed from
+the value of the `-DomainName` parameter.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:191]:
```suggestion
If this parameter is set with a valid NetBIOS name value, then forest installation continues with
the name specified. If this parameter is not set, then the default is automatically computed from
the value of the **DomainName** parameter.
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 6]
```diff
@@ -154,19 +182,24 @@ Accept wildcard characters: False
...
 
-For example, if this parameter is not specified and a single-label prefix domain name of 15 characters or less is specified within the value of the *DomainName* parameter, then promotion continues with an automatically generated NetBIOS domain name.
-For example, the prefix label corp within a full domain name value of corp.contoso.com would be a successful name choice.
+For example, if this parameter is not specified and a single-label prefix domain name of 15
+characters or less is specified within the value of the `-DomainName` parameter, then promotion
+continues with an automatically generated NetBIOS domain name. For example, the prefix label corp
+within a full domain name value of corp.contoso.com would be a successful name choice.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:196]:
```suggestion
For example, if this parameter is not specified and a single-label prefix domain name of 15
characters or less is specified within the value of the **DomainName** parameter, then promotion
continues with an automatically generated NetBIOS domain name. For example, the prefix label `corp`
within a full domain name value of `corp.contoso.com` would be a successful name choice.
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 7]
```diff
@@ -193,9 +227,10 @@ Accept wildcard characters: False
 ```
 
 ### -ForestMode
-Specifies the forest functional level for the new forest.
-Supported values for this parameter can be either a valid integer or a corresponding enumerated string value.
-For example, to set the forest mode level to Windows Server 2008 R2, you can specify either a value of 4 or Win2008R2.
+
+Specifies the forest functional level for the new forest. Supported values for this parameter can be
+either a valid integer or a corresponding enumerated string value. For example, to set the forest
+mode level to Windows Server 2008 R2, you can specify either a value of 4 or Win2008R2.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:233]:
```suggestion
Specifies the forest functional level for the new forest. Supported values for this parameter can
be either a valid integer or a corresponding enumerated string value. For example, to set the
forest mode level to Windows Server 2008 R2, you can specify either a value of `4` or `Win2008R2`.
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 8]
```diff
@@ -206,11 +241,12 @@ The acceptable values for this parameter are:
 - Windows Server 2012 R2:  6 or Win2012R2
 - Windows Server 2016: 7 or WinThreshold
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:242]: I can't comment on the rest of the lines here (and other instances of this throughout the file), but if you can ensure the specific valid values for the parameters are wrapped in backticks, that would be great!

```suggestion
- Windows Server 2012 R2:  `6` or `Win2012R2`
- Windows Server 2016: `7` or `WinThreshold`
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 9]
```diff
@@ -206,11 +241,12 @@ The acceptable values for this parameter are:
...
 
-The default forest functional level in Windows Server is typically the same as the version you are running.
-However, the default forest functional level in Windows Server 2008 R2 when you create a new forest is Windows Server 2003 or 2.
+The default forest functional level in Windows Server is typically the same as the version you are
+running. However, the default forest functional level in Windows Server 2008 R2 when you create a
+new forest is Windows Server 2003 or 2.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:246]:
```suggestion
The default forest functional level in Windows Server is typically the same as the version you are
running. However, the default forest functional level in Windows Server 2008 R2 when you create a
new forest is Windows Server 2003 or `2`.
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 10]
```diff
@@ -223,11 +259,12 @@ Accept wildcard characters: False
 ```
 
 ### -InstallDns
+
 Indicates that this cmdlet installs and configures the DNS Server service for the new forest.
 For forest installation, the default is $True.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:264]:
```suggestion
Indicates that this cmdlet installs and configures the DNS Server service for the new forest.
For forest installation, the default is `$true`.
```

File: docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md [hunk 11]
```diff
@@ -333,13 +382,17 @@ Accept wildcard characters: False
...
-When this switch parameter is set, it specifies that additional preliminary checks should be bypassed.
-For more information on the scope of these additional preliminary checks that the ADDSDeployment module performs by default when using Windows Server 2012, refer to the table in the section ADPrep and Prerequisite Checking Architecture in [AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
+
+Indicates that the cmdlet performs only a base set of validations. This behavior is equivalent to
+the validations that were performed when using `Dcpromo.exe` in earlier versions of Windows Server
+to add a new forest. When this switch parameter is set, it specifies that additional preliminary
+checks should be bypassed. For more information on the scope of these additional preliminary checks
+that the ADDSDeployment module performs by default when using Windows Server 2012, refer to the
+table in the section ADPrep and Prerequisite Checking Architecture in
+[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Install-ADDSForest.md:392]:
```suggestion
Indicates that the cmdlet performs only a base set of validations. This behavior is equivalent to
the validations that were performed when using `Dcpromo.exe` in earlier versions of Windows Server
to add a new forest. When this switch parameter is set, it specifies that additional preliminary
checks should be bypassed. For more information on the scope of these additional preliminary checks
that the **ADDSDeployment** module performs by default when using Windows Server 2012, refer to the
table in the section ADPrep and Prerequisite Checking Architecture in
[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**